### ISSUE TITLE
Possible to set LED level on some other buttons

### ISSUE THREAD

**Issue by riban-bw** [giezu/LaunchkeyMiniMK3#2]
There are 4 buttons that can have their white LED turned on at various levels:

- Launch (CC 104)
- Stop / Solo / Mute (CC 105)
- Play (CC 115)
- Record (CC 117)

Each may be illuminated by sending CC with value parameter defining brightness:

- 0-31 0% (off)
- 32-47 25%
- 48-63 50%
- 64-95 75%
- 96-127 100% (full on)

**Comment by giezu** [comment]
Yes! I remember that Stop/Solo/Mute can be colored to yellow/blue/red but I forgot to investigate that further!
I will check that tomorrow, should I add that info and credit you like before or you will make PR?

**Comment by riban-bw** [comment]
You can just add it. I can't be bothered to fork, edit, PR. I have loads of forks I need to tidy up for simple PRs like this. (There must be a better way!!!)

I haven't figured out how to change colours of the buttons. I notice the 25% illumination looks purple but that might just be how they encode the brightness.

Thanks for investigating and maintaining this repo. It is proving very useful in my integration work hence feedback to you of anything else I find so that other's may also benefit.

**Comment by riban-bw** [comment]
All the colours in the colour chart seem to be valid, e.g. sending value 95 will light the button hot pink.

**Comment by giezu** [comment]
Great!

**Comment by riban-bw** [comment]
You only show 2 examples for colouring buttons which might confuse as there are 3 modes: solid, pulsing & flashing.

You haven't mentioned being able to set these 3 modes for play / record buttons.

**Comment by giezu** [comment]
Added more examples
